src/features/locators/styles/locatorsList.less

```diff
@@ -222,8 +222,10 @@
     }
 
     &_search {
-      width: 264px;
+      width: 275px;
+      height: 24px;
       margin-top: 8px;
+      padding: 0 11px;
```

Comment:
please, have a look at this example https://4x.ant.design/components/input/#components-input-demo-size
antd's Input has a `size` property, probably, it will be enough just to add `size="small"`